Engine/AssemblyFinder.cs

```diff
@@ -97,6 +97,8 @@ private void SyncFiles()
                             {
                                 if (StrEq(subDir.Name, "obj"))
                                     continue; // skip obj subfolder
+                                if (subDir.GetFiles("OpenTap.dll").Any())
+                                    continue; // skip subdirectory if it appears to be an opentap installation
```

Comment:
We should probably log a warning when this happens the first time for a given directory.